src/main/java/com/pgs/booking/validators/UserValidator.java

```diff
@@ -4,7 +4,7 @@
 import com.pgs.booking.model.entity.Role;
 import com.pgs.booking.model.entity.User;
 import lombok.AllArgsConstructor;
-import org.springframework.security.web.authentication.preauth.PreAuthenticatedAuthenticationToken;
```

Comment:
why removed as method contains this class `PreAuthenticatedAuthenticationToken`

Reply:
Endpoint did not work properly. There was a problem with checking the active User. The change was forced by Peter's changes.